runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/FlinkBatchTransformTranslators.java

```diff
@@ -489,6 +489,14 @@ public void translateNode(
     }
   }
 
+  private static void rejectSplittable(DoFn<?, ?> doFn) {
+    DoFnSignature signature = DoFnSignatures.getSignature(doFn.getClass());
+    if (signature.processElement().isSplittable()) {
+      throw new UnsupportedOperationException(
+          String.format("FlinkRunner does not currently support Splittable DoFn: %s", doFn));
```

Comment:
Error message nits:

 - For proper identifiers, interpolate when possible, as in `FlinkRunner.class.getSimpleName()`
 - `Splittable` shouldn't be capitalized. It is just a `splittable DoFn`.